Classes/Login/LoginViewController.swift

```diff
@@ -12,12 +12,14 @@ import OnePasswordExtension
 final class LoginViewController: UITableViewController, UITextFieldDelegate {
 
     var client: GithubClient!
+    var isInitialLogin: Bool = true
 
     let signinCellIndexPath = IndexPath(item: 0, section: 2)
 
     @IBOutlet weak var usernameTextField: UITextField!
     @IBOutlet weak var passwordTextField: UITextField!
     @IBOutlet weak var onepasswordButton: UIButton!
+    @IBOutlet var cancelButton: UIBarButtonItem!
```

Comment:
This `@IBOutlet` is not `weak`, as it would be released when setting `rightBarButtonItem` to `nil`, thus not re-adding when calling `showCancelButton(true)`.